src/kirby/components/card/card.component.tns.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../scss/native-imports';
+
+$color-card-bg: rgb(255, 255, 255);
+$color-card-border: rgb(133, 133, 133);
+.background {
```

Comment:
Kunne være .background, .container {... i stedet for. Så kan du spare at sætte de samme tre properties i .container ;)

Reply:
Fixed